integration/users/update_user_test.go

```diff
@@ -242,8 +242,6 @@ func TestUpdateUser(t *testing.T) {
 				setup.SkipForMongoDB(t, tc.skipForMongoDB)
 			}
 
-			t.Parallel()
```

Comment:
Any reason we can't run these tests in parallel? :thinking:

Reply:
Hey Chi! Adding it back. This shouldn't be removed from the pull request. I removed while trying to debug something else.